gym/spaces/discrete.py

```diff
@@ -1,8 +1,10 @@
+from typing import Optional
+
 import numpy as np
 from .space import Space
 
 
-class Discrete(Space):
+class Discrete(Space[int]):
```

Comment:
Is this actually correct? `space.sample()` will return a np.int64 by default, and from a quick check it seems that this might cause type checkers to throw some false warnings. Can you double check if this will actually work in terms of subtypes etc between python `int` and numpy `int`s?

Reply:
As far as I can tell, Discrete produces native int on sample method now:
```python
Python 3.8.8 (default, Apr 13 2021, 12:59:45) 
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from gym.spaces import Discrete
>>> d = Discrete(3)
>>> type(d.sample())
<class 'int'>
```

Reply:
Correct, my bad